Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -480,10 +497,8 @@ - (void)_updateScreenTimeBasedOnWindowVisibility
     BOOL showsSystemScreenTimeBlockingView = [_configuration showsSystemScreenTimeBlockingView];
 
     if (viewIsInWindow) {
-        BOOL previouslyInstalledScreenTimeWebpageController = !!_screenTimeWebpageController;
         [self _installScreenTimeWebpageControllerIfNeeded];
-        if (!previouslyInstalledScreenTimeWebpageController && _screenTimeWebpageController)
-            [_screenTimeWebpageController setURL:[self _mainFrameURL]];
+        [_screenTimeWebpageController setURL:[self _mainFrameURL]];
```

Comment:
I don't quite follow the reasoning for the old logic; make sure it wasn't important?